src/main/java/com/lucidworks/spark/streaming/netty/HttpClientHandler.java

```diff
@@ -3,11 +3,7 @@
 import com.lucidworks.spark.streaming.MessageHandler;
```

Comment:
I think we can delete this streaming netty-based package ... it's not really being used.